test/metabase/api/card_test.clj

```diff
@@ -680,7 +680,7 @@
 
 ;; Test that we *cannot* share a Card if the Card has been archived
 (expect
-  "The object has been archived."
+  {:message "The object has been archived.", :error_code "archived"}
```

Comment:
I don't think `error_code` is a pattern we've used before in any endpoints. I'm not opposed to it or similar, but we should run it by @camsaul.